app/components/provider_interface/diversity_information_component.rb

```diff
@@ -9,32 +9,21 @@ def initialize(application_choice:, current_provider_user:)
       @current_provider_user = current_provider_user
     end
 
-    def message
-      return declared_diversity_information_message if diversity_information_declared?
-
-      'No information shared'
-    end
-
     def display_diversity_information?
       diversity_information_declared? && current_user_has_permission_to_view_diversity_information? && application_in_correct_state?
     end
 
-    def details
-      if [current_user_has_permission_to_view_diversity_information?, application_in_correct_state?].all?(false)
-        return "Users with permission to see this information will only be able to do so after #{offer_context} offer has been accepted by the candidate."
+    def rows
+      rows = [{ key: 'Do you want to answer a few questions about your sex, disability and ethnicity?', value: diversity_information_declared? ? 'Yes' : 'No' }]
+
+      if diversity_information_declared?
+        rows << { key: 'What is your sex?', value: row_value(equality_and_diversity['sex'].capitalize) }
```

Comment:
Did you try grabbing these keys from the same ‘resource’ as the candidate form labels?

This way they will just stay in sync.